google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -429,6 +444,9 @@ public void setDevAppserverLogLevel(String devAppserverLogLevel) {
 
   @Override
   public Boolean getSkipSdkUpdateCheck() {
+    if (settings.isSkipSdkUpdateCheck() == null) {
```

Comment:
Yeah, that approach works.  I've changed everything to initialize in the settings class.